group_vars/location_segen/snmp.yml

```diff
@@ -0,0 +1,39 @@
+---
+
+snmp_devices:
+  - hostname: segen-saarbr
+    address: 10.31.6.10
+    snmp_profile: airos_8
```

Comment:
airos 8?

Reply:
Please create an SNMP Profile for GIGABEAM LR  and use it.